python/setup.py

```diff
@@ -65,6 +66,11 @@ def get_version(pkg_path):
     extra_link_args = []
     extra_define_macros = []
 
+# Setting the CFLAGS environment variable doesn't seem sufficient in all cases to add
```

Comment:
Do you know where this limitation was coming into play? I've generally always had success with CFLAGS, unless activating a conda environment with a compilers package that may overwrite this. I don't _think_ that is the problem here but figured I'd mention